c2corg_api/models/mailinglist.py

```diff
@@ -0,0 +1,35 @@
+from sqlalchemy import Column, DateTime, ForeignKey, Integer, String
+from sqlalchemy.orm import relationship
+from sqlalchemy.sql.functions import func
+
+from c2corg_api.models import Base, sympa_schema, users_schema
+from c2corg_api.models.user import User
+
+
+class Mailinglist(Base):
+    """
+    Class containing mailing list subscriptions.
+    Based upon Sympa mailing list server https://www.sympa.org/
+    """
+    __tablename__ = 'subscriber_table'
+    __table_args__ = {"schema": sympa_schema}
+
+    listname = Column('list_subscriber', String(50), primary_key=True)
+    email = Column('user_subscriber', String(200), primary_key=True)
+    user_id = Column(
```

Comment:
That makes sense. I think I have used ``listname`` + ``email`` as a composite primary key because that was the one used in v5:
```
                        Table "public.subscriber_table"
           Column           |           Type           |       Modifiers        
----------------------------+--------------------------+------------------------
 list_subscriber            | character varying(50)    | not null
 user_subscriber            | character varying(100)   | not null
 date_subscriber            | timestamp with time zone | not null default now()
 update_subscriber          | timestamp with time zone | 
 visibility_subscriber      | character varying(20)    | 
 reception_subscriber       | character varying(20)    | 
 bounce_subscriber          | character varying(35)    | 
 bounce_score_subscriber    | integer                  | 
 comment_subscriber         | character varying(150)   | 
 subscribed_subscriber      | smallint                 | 
 included_subscriber        | smallint                 | 
 include_sources_subscriber | character varying(50)    | 
Indexes:
    "ind_subscriber" PRIMARY KEY, btree (list_subscriber, user_subscriber)
    "subscriber_idx" btree (user_subscriber, list_subscriber)
```
By the way we also have to take into account the fact that this table is used by another system (Sympa) that ignores the ``user_id`` and for which the "email" primary key is more relevant.
What do you think?

There's also a "reverse" index (user_subscriber, list_subscriber). I don't know if it is really needed. For instance because we don't use Sympa for writing in the table and because the table is not that big (~5000 rows as for now) so perhaps indices are not that critical?